invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/InvitationActionButtons.js

```diff
@@ -9,30 +9,50 @@
 import React from "react";
 import { Button } from "semantic-ui-react";
 import { i18next } from "@translations/invenio_communities/i18next";
+import _isEmpty from "lodash/isEmpty";
+import PropTypes from "prop-types";
 
-export const ViewButton = ({ onClick }) => (
-  <Button size="tiny" content={i18next.t("View")} onClick={onClick} />
-);
+export const ViewButton = ({ onClick, show }) =>
+  show && <Button size="tiny" content={i18next.t("View")} onClick={onClick} />;
 
-export const CancelButton = ({ onClick }) => (
-  <Button size="tiny" content={i18next.t("Cancel")} onClick={onClick} />
-);
+export const CancelButton = ({ onClick, show }) =>
+  show && (
+    <Button size="tiny" content={i18next.t("Cancel")} onClick={onClick} />
+  );
 
-export const ReInviteButton = ({ onClick }) => (
-  <Button content={i18next.t("Re invite")} onClick={onClick} />
-);
+export const ReInviteButton = ({ onClick, show }) =>
+  show && (
+    <Button size="tiny" content={i18next.t("Re invite")} onClick={onClick} />
+  );
 
-export const ActionButtons = ({ status, onReInvite, onView, onCancel }) => {
+export const ActionButtons = ({
+  actions,
+  onReInvite,
+  onView,
+  onCancel,
+  status,
+}) => {
+  status = status.toLowerCase();
   const condition = false;
+  const actionsPossible = !_isEmpty(actions);
 
-  if (condition) {
-    return <ReInviteButton onClick={() => onReInvite()} />;
-  }
+  const showViewButton = true;
+  const showCancel = status === "submitted" && actionsPossible;
```

Comment:
where are the statuses defined?

Reply:
Good question, i tried to use the statuses defined in the UX RFC https://codimd.web.cern.ch/f3a7_DoqSfed6m_VMa649A#List-invitations but they seem to be different from what is actually being used right now

Reply:
This is still to be fixed.

Reply:
As in the statuses, or the implementation?

Reply:
The statuses, maybe a chat with @kpsherva ?